Sources/Kommunicate/Classes/Kommunicate.swift

```diff
@@ -487,6 +487,7 @@ open class Kommunicate: NSObject,Localizable{
                             completion(KommunicateError.conversationNotPresent)
                             return
                         }
+                        ALMessageService.syncMessages()
```

Comment:
Should we not do it inside the controller when the view is visible or loaded?

Reply:
Added the changes.